plexpy/monitor.py

```diff
@@ -0,0 +1,438 @@
+﻿# This file is part of PlexPy.
```

Comment:
This class/file no longer exists. It's been split up into `activity_pinger` and `activity_processor`.